libs/brain/switch/src/lib/brn-switch.component.ts

```diff
@@ -36,48 +35,46 @@ export const BRN_SWITCH_VALUE_ACCESSOR = {
 
 const CONTAINER_POST_FIX = '-switch';
 
+let uniqueIdCounter = 0;
+
 @Component({
 	selector: 'brn-switch',
-	imports: [NgStyle],
+	standalone: true,
```

Comment:
I think this can be removed now that we target Angular v19+ this is the default.